shop_app.rb

```diff
@@ -0,0 +1,47 @@
+require "bundler/setup"
+require "sinatra/base"
+require "sinatra/reloader"
+
+Dir["./lib/**/*.rb"].each { |ruby_file| require ruby_file }
+require "./db/seeds.rb"
+
+module Shop
+  class App < Sinatra::Base
```

Comment:
Project directory has `shop` name and this name is not important here. Inside project directory, at the root level you have `shop_app.rb` file with `module Shop; class App` inside. By convention it should be placed in `shop/app.rb` (`shop` inside your project's directory where you have git repository).